piccolo_api/media/local.py

```diff
@@ -95,7 +95,7 @@ def save():
             with open(path, "wb") as new_file:
                 shutil.copyfileobj(file, new_file)
                 if file_permissions is not None:
-                    os.chmod(path, 0o640)
+                    os.chmod(path, 0o600)
```

Comment:
I think we have a mistake here (it was pre-existing). Should be:

```python
if file_permissions is not None:
    os.chmod(path, file_permissions)
```

Reply:
Great catch. `file_permissions` are already defined in constructor. I didn't notice it at all.